### ISSUE TITLE
[Bug] FLA models fail to return logits when labels are not provided

### ISSUE THREAD

**Issue by sustcsonglin** [fla-org/flash-linear-attention#237]
### Checklist

- [x] I have checked [FAQs](https://github.com/fla-org/flash-linear-attention/blob/main/FAQs.md) and existing issues for similar problems
- [x] Please report this bug in English to ensure wider understanding and support

### Describe the Bug

The `fuse_cross_entropy` option is enabled by default. In the following code segment:  
[modeling_transformer.py#L373C1-L374C1](https://github.com/fla-org/flash-linear-attention/blob/main/fla/models/transformer/modeling_transformer.py#L373C1-L374C1), there appears to be a logic issue that prevents logits from being returned when no labels are provid

### Steps to Reproduce the Bug
N/A
### Expected Behavior
N/A

### Environment Information

should be applied for all version
